pkgs/development/python-modules/slack-bolt/default.nix

```
@@ -32,6 +32,11 @@ buildPythonPackage rec {
     hash = "sha256-s9djd/MDNnyNkjkeApY6Fb1mhI6iop8RghaSJdi4eAs=";
   };
 
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace "pytest-runner==5.2" ""
+  '';
```

```suggestion
  # The packaged pytest-runner version is too new as of 2023-07-27. It's not really needed anyway. Unfortunately, 
  # pythonRelaxDepsHook doesn't work on setup_requires packages.
  postPatch = ''
    substituteInPlace setup.py --replace "pytest-runner==5.2" ""
  '';
```

for some reason this only seems to be necessary on linux. might be worth considering gating with `lib.optionals stdenv.isLinux`... I could go either way on that

---

Sandro told me to avoid branching patches as much as possible.
If it doesn't break darwin, I'll leave it as is.